packages/roosterjs-content-model-types/lib/editor/EditorPlugin.ts

```diff
@@ -42,4 +52,9 @@ export interface EditorPlugin {
      * @param event The event to handle:
      */
     onPluginEvent?: (event: PluginEvent) => void;
+
+    /**
+     * @returns The content model configuration for this plugin
+     */
+    getContentModelConfig?: () => PluginContentModelConfig;
```

Comment:
Add comment to explain what it does. And need to say this method will be called before initialize(), so we can't use editor object in this method.